crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -354,37 +223,157 @@ pub enum Behavior {
 }
 ```
 
-### Deprecate a rule
+### Coding the rule
 
-There are occasions when a rule must be deprecated, to avoid breaking changes. The reason
-of deprecation can be multiple.
+Below, there are many tips and guidelines on how to create a lint rule using Biome infrastructure.
 
-In order to do, the macro allows adding additional field to add the reason for deprecation
+#### Navigating the CST
```

Comment:
We previously used the terminology `AST`. We should stick with this terminology?
```suggestion
#### Navigating the AST
```

Reply:
I think we should change to CST (maybe in another PR)